<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SASUPPORT13" ignored="1" name="Default test(failed)" tests="2" failures="1" timestamp="16 Sep 2020 11:01:51 GMT" time="0.057" errors="0">
  <testcase name="test2" time="0.012" classname="re_run_failed_test.Re_run_failedTest"/>
  <testcase name="test3" time="0.011" classname="re_run_failed_test.Re_run_failedTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at re_run_failed_test.Re_run_failedTest.test3(Re_run_failedTest.java:23)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test1" time="0.0" classname="re_run_failed_test.Re_run_failedTest">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
